scss/dijit/form/_button.scss

```diff
@@ -1,267 +1,278 @@
 
-        &:before {
-          @if $combobutton-focused-arrow-url {
-            background-image: url($combobutton-focused-arrow-url);
-          } @else {
-            @include create-arrow($combobutton-arrow-size, $combobutton-focused-arrow-color);
-          }
+                &:before {
+                    @if $combobutton-disabled-arrow-url {
+                        background-image: url($combobutton-disabled-arrow-url);
+                    } @else {
+                        @include create-arrow($combobutton-arrow-size, $combobutton-disabled-arrow-color);
+                    }
+                }
+            }
         }
-      }
     }
 
-    &Disabled {
-      @include btn($combobutton-disabled-shadow, $combobutton-disabled-bgcolor, $combobutton-disabled-border, $combobutton-disabled-textcolor);
+    &ToggleButton {
+        @include btn($togglebutton-normal-shadow, $togglebutton-normal-bgcolor, $togglebutton-normal-border, $togglebutton-normal-textcolor);
 
-      .dijitArrowButtonInner {
-        border-left: $combobutton-disabled-arrow-border;
-
-        @if $combobutton-disabled-arrow-bgcolor != 'transparent' {
-          background-color: $combobutton-disabled-arrow-bgcolor;
+        .dijitIcon {
+            visibility: hidden;
         }
 
-        &:before {
-          @if $combobutton-disabled-arrow-url {
-            background-image: url($combobutton-disabled-arrow-url);
-          } @else {
-            @include create-arrow($combobutton-arrow-size, $combobutton-disabled-arrow-color);
-          }
+        &Checked {
+            .dijitIcon {
+                visibility: visible;
+            }
         }
-      }
-    }
-  }
-
-  &ToggleButton {
-    @include btn($togglebutton-normal-shadow, $togglebutton-normal-bgcolor, $togglebutton-normal-border, $togglebutton-normal-textcolor);
 
-    .dijitIcon {
-      visibility: hidden;
-    }
+        &Focused,
+        &Checked {
+            @include btn($togglebutton-active-shadow, $togglebutton-active-bgcolor, $togglebutton-active-border, $togglebutton-active-textcolor);
+        }
 
-    &Checked {
-      .dijitIcon {
-        visibility: visible;
-      }
-    }
+        &Hover,
+        &HoverFocused,
+        &CheckedHover {
+            @include btn($togglebutton-hover-shadow, $togglebutton-hover-bgcolor, $togglebutton-hover-border, $togglebutton-hover-textcolor);
+        }
 
-    &Focused,
-    &Checked {
-      @include btn($togglebutton-active-shadow, $togglebutton-active-bgcolor, $togglebutton-active-border, $togglebutton-active-textcolor);
+        &Disabled {
+            @include btn($togglebutton-disabled-shadow, $togglebutton-disabled-bgcolor, $togglebutton-disabled-border, $togglebutton-disabled-textcolor);
+        }
     }
+}
 
-    &Hover,
-    &HoverFocused,
-    &CheckedHover {
-      @include btn($togglebutton-hover-shadow, $togglebutton-hover-bgcolor, $togglebutton-hover-border, $togglebutton-hover-textcolor);
-    }
+.listingTable table {
```

Comment:
comboboxes can be wrapper in a `table` because of this rule: https://github.com/dotCMS/core/blob/2dd33fa42efeb7de9bf425f274db7acd793cdd3a/dotCMS/src/main/webapp/html/css/dijit-dotcms/dotcms.css#L7214